files/es/web/api/indexeddb/index.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: WindowOrWorkerGlobalScope.indexedDB
+title: indexedDB
 slug: Web/API/indexedDB
 original_slug: Web/API/WindowOrWorkerGlobalScope/indexedDB
```

Comment:
Hey, @Graywolf9 thanks for contributing, should the original slug still calling the window or worker global scope?

Reply:
Thanks for the review! I had the same doubt but there is other issue about the front matter keys that says that it is a front matter added from Yari so it updates as necessary for it

Reply:
but even that didn't you think we should remove it to avoid confusion in possible next reviews?

Reply:
https://github.com/mdn/translated-content/issues/5617#issuecomment-1153824408